### PR TITLE
Fix build on FreeBSD

### PR DESCRIPTION
**Pull request by MikaelUrankar**

### CODE REVIEW

File: cli/upgrade.rs [hunk 1]
```diff
@@ -36,6 +36,8 @@ const ARCHIVE_NAME: &str = "deno-x86_64-pc-windows-msvc.zip";
 const ARCHIVE_NAME: &str = "deno-x86_64-apple-darwin.zip";
 #[cfg(target_os = "linux")]
 const ARCHIVE_NAME: &str = "deno-x86_64-unknown-linux-gnu.zip";
+#[cfg(target_os = "freebsd")]
+const ARCHIVE_NAME: &str = "deno-x86_64-unknown-freebsd.zip";
```
Comment (tokiedokie) [inline cli/upgrade.rs:40]:
```suggestion
const ARCHIVE_NAME: &str = "deno-x86_64-unknown-freebsd.zip";
```

this `ARCHIVE_NAME` is the target zip file name used in https://github.com/denoland/deno/releases
so we have to create a binary for freebsd

Comment (MikaelUrankar) [reply cli/upgrade.rs:40]: I'm not sure how easy it'll be for you. We have a port in our infrastructure if that helps you: https://github.com/freebsd/freebsd-ports/tree/master/www/deno

Comment (ry) [reply cli/upgrade.rs:40]: We need a FreeBSD build in CI to support it in upgrade. Adding this will just introduce errors.

File: cli/ops/tty.rs [hunk 2]
```diff
@@ -311,7 +311,7 @@ fn op_console_size(
         let fd = std_file.as_raw_fd();
         unsafe {
           let mut size: libc::winsize = std::mem::zeroed();
-          if libc::ioctl(fd, libc::TIOCGWINSZ, &mut size as *mut _) != 0 {
+          if libc::ioctl(fd, libc::TIOCGWINSZ.into(), &mut size as *mut _) != 0 {
```
Comment (ry) [inline cli/ops/tty.rs:314]: This shouldn't be necessary

**Comment by ry** [comment]
Glad you have the freebsd port working https://github.com/freebsd/freebsd-ports/tree/master/www/deno

 I don't think either of these changes are need tho.

**Pull request closed without merging**